documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -44,10 +44,19 @@ To use the functions, add ``#include <oneapi/dpl/iterator>`` to your code. For e
   #include <oneapi/dpl/iterator>
   #include <sycl/sycl.hpp>
   int main(){
-    sycl::buffer<int> buf { 1000 };
+    std::vector<int> vec(1000);
+
+    std::random_device r;
+    std::default_random_engine e1(r());
+    std::uniform_int_distribution<int> uniform_dist(-2000, 2000);
+    std::generate(vec.begin(), vec.end(), [&](){ return uniform_dist(e1); });
```

Comment:
I think this is quite overcomplicated for the examples. I would simplify it to just
```suggestion
    std::generate(vec.begin(), vec.end(), std::minstd_rand{});
```
Also `<random>` should be included.

Reply:
Thanks, done.